documentation/modules/configuring/con-maintenance-time-window-definition.adoc

```diff
@@ -6,13 +6,38 @@
 = Scheduling maintenance time windows
 
 [role="_abstract"]
-Schedule Cluster Operator certificate renewals for Kafka clusters to minimize impact on client applications.
-Use time windows in conjunction with the xref:con-certificate-renewal-str[renewal periods of the CA certificates created by the Cluster Operator] (`Kafka.spec.clusterCa.renewalDays` and `Kafka.spec.clientsCa.renewalDays`).
+Use maintenance time windows to control when the Cluster Operator performs certificate renewals and rolling restarts.
+This helps minimize disruption to Kafka clients by scheduling updates at convenient times.
 
-Updates are usually triggered by changes to the `Kafka` resource by the user or through user tooling.
-Rolling restarts for certificate expiration may occur without `Kafka` resource changes.
-While unscheduled restarts shouldn't affect service availability, they could impact the performance of client applications.
-Maintenance time windows allow scheduling of these updates for convenient times.
+Maintenance time windows apply only to automatic certificate renewals and rolling restarts managed by the Cluster Operator.
+
+They **do** apply in the following scenarios:
+
+* Automatic renewal of Strimzi-managed internal certificates, such as those used by Kafka nodes.
```

Comment:
`Automatic` might be confusing here. Maybe we should fold it under the following point as this would apply only to the time-based renewals?